DNN Platform/Library/Entities/Tabs/TabController.cs

```diff
@@ -2735,15 +2735,29 @@ public static XmlNode SerializeTab(XmlDocument tabXml, TabInfo objTab, bool incl
                     urlNode.Attributes.Append(XmlUtils.CreateAttribute(tabXml, "type", "Normal"));
                     break;
                 case TabType.Tab:
-                    urlNode.Attributes.Append(XmlUtils.CreateAttribute(tabXml, "type", "Tab"));
-                    //Get the tab being linked to
-                    TabInfo tempTab = TabController.Instance.GetTab(Int32.Parse(tab.Url), tab.PortalID, false);
-                    urlNode.InnerXml = tempTab.TabPath;
+                    int tabId;
+                    if (Int32.TryParse(tab.Url, out tabId))

```

Comment:
Can you switch out `Int32` for the `int` keyword/alias (both here and below)?  Thanks!